ejabberd.yml

```diff
@@ -0,0 +1,510 @@
+## accounts.
+##
+## watchdog_admins:
+##   - "admin@dev-ejabberd.qa.skillz.com"
+
+###.  ===============
+###'  NODE PARAMETERS
+
+##
+## net_ticktime: Specifies net_kernel tick time in seconds. This options must have
+## identical value on all nodes, and in most cases shouldn't be changed at all from
+## default value.
+##
+## net_ticktime: 60
+
+###.  ================
+###'  SERVED HOSTNAMES
+
+##
+## hosts: Domains served by ejabberd.
+## You can define one or several, for example:
+## hosts:
+##   - "example.net"
+##   - "example.com"
+##   - "example.org"
+##
+hosts:
+  - "dev-ejabberd"
+  - "localhost"
+  - "192.168.22.37"
+
+##
+## route_subdomains: Delegate subdomains to other XMPP servers.
+## For example, if this ejabberd serves example.org and you want
+## to allow communication with an XMPP server called im.example.org.
+##
+## route_subdomains: s2s
+
+###.  ===============
+###'  LISTENING PORTS
+
+##
+## listen: The ports ejabberd will listen on, which service each is handled
+## by and what options to start it with.
+##
+listen:
+  - 
+    port: 5222
+    module: ejabberd_c2s
+    shaper: c2s_shaper
+    access: c2s
+    resend_on_timeout: if_offline
+    starttls: false
+    starttls_required: false
+  -
+    port: 5269
+    module: ejabberd_s2s_in
+    max_stanza_size: 131072
+    shaper: s2s_shaper
+    s2s_use_starttls: false
+  - 
+    port: 5280
+    module: ejabberd_http
+    request_handlers:
+      "/websocket": ejabberd_http_ws
+    http_bind: true
+    captcha: false
+    tls: false
+  -
+    port: 5290
+    module: ejabberd_http
+    request_handlers:
+      "/api": mod_http_api
+    http_bind: true
+
+##.  ==============
+##'  AUTHENTICATION
+
+#
+# auth_method: Method used to authenticate the users.
+# The default method is the internal.
+# If you want to use a different method,
+# comment this line and enable the correct ones.
+#
+
+auth_method: http
+auth_opts:
+  host: "http://localhost:9100"
+  connection_pool_size: 8
+  connection_opts: []
+  basic_auth: "chat:zA3B7HGP83x99MoKvRXR967fk8"
+  path_prefix: "/chat/v1/auth/"
+
+auth_password_format: plain
+disable_sasl_mechanisms:
+  - "digest-md5"
+  - "scram-sha-1"
+
+default_db: sql
+
+sql_type: mysql
+sql_server: "127.0.0.1"
+
+sql_port: 3306
+sql_database: "ejabberd"
+sql_username: "ejabberd"
+sql_password: "ejabberd"
+sql_keepalive_interval: 60
+sql_start_interval: 5
+sql_pool_size: 10
+
+## Define the FQDN if ejabberd doesn't detect it:
+## fqdn: "server3.example.com"
+
+###.  ===============
+###'  TRAFFIC SHAPERS
+
+##
+## This option specifies the maximum number of elements in the queue
+## of the FSM. Refer to the documentation for details.
+##
+max_fsm_queue: 1000
+
+###.   ====================
+###'   ACCESS CONTROL LISTS
+acl:
+  ##
+  ## The 'admin' ACL grants administrative privileges to XMPP accounts.
+  ## You can put here as many accounts as you want.
+  ##
+  admin:
+    user:
+      - "admin": "chat.dev.skillz.com"
```

Comment:
also add `admin` and `skillz-cas` for `dev-ejabberd`

Reply:
done